packages/server/src/api/controllers/view/viewsV2.ts

```diff
@@ -127,13 +127,13 @@ export async function create(ctx: Ctx<CreateViewRequest, ViewResponse>) {
 
   const parsedView: Omit<RequiredKeys<ViewV2>, "id" | "version"> = {
     name: view.name,
+    type: view.type,
     tableId: view.tableId,
     query: view.query,
     queryUI: view.queryUI,
     sort: view.sort,
     schema,
     primaryDisplay: view.primaryDisplay,
-    uiMetadata: view.uiMetadata,
```

Comment:
Is this meant to be removed?

Reply:
Yes. Its only purpose prior to this was for the frontend to identify calculation views. This is now possible with `type`.